docs/_build_old/html-v1-0-0/index.html

```diff
@@ -0,0 +1,309 @@
+      <nav class="wy-nav-top" aria-label="top navigation">
+        
+          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
+          <a href="#">zstash</a>
+        
+      </nav>
+
+
+      <div class="wy-nav-content">
+        
+        <div class="rst-content">
+        
+          
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<div role="navigation" aria-label="breadcrumbs navigation">
+
+  <ul class="wy-breadcrumbs">
+    
+      <li><a href="#" class="icon icon-home"></a> &raquo;</li>
+        
+      <li>Zstash documentation</li>
+    
+    
+      <li class="wy-breadcrumbs-aside">
+        
+            
+            <a href="_sources/index.rst.txt" rel="nofollow"> View page source</a>
+          
+        
+      </li>
+    
+  </ul>
+
+  
+  <hr/>
+</div>
+          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
+           <div itemprop="articleBody">
+            
+  <div class="section" id="zstash-documentation">
+<span id="index-label"></span><h1>Zstash documentation<a class="headerlink" href="#zstash-documentation" title="Permalink to this headline">¶</a></h1>
+<div class="section" id="what-is-zstash">
+<h2>What is zstash?<a class="headerlink" href="#what-is-zstash" title="Permalink to this headline">¶</a></h2>
+<p>Zstash is an <strong>HPSS long-term archiving</strong> solution for E3SM.</p>
+<p>Zstash is written entirely in Python using standard libraries.
+Its design is intentionally minimalistic to provide an effective
+long-term HPSS archiving solution without creating an overly complicated
+(and hard to maintain) tool.</p>
+<p><strong>Key features:</strong></p>
+<ul class="simple">
+<li><p>Files are archived into standard <strong>tar files</strong> with a user <strong>specified maximum size</strong>.</p></li>
+<li><p>Tar files are first created locally, then transferred to HPSS.</p></li>
+<li><p><strong>Checksums (md5)</strong> of input files are computed <em>on-the-fly</em> during
+archiving. For large files, this saves a considerable amount of
+time compared to separate checksumming and archiving steps.</p></li>
+<li><p>Checksums and additional metadata (size, modification time, tar file and offset)
+are stored in a sqlite3 <strong>index database</strong>.</p></li>
+<li><p><strong>Database enables faster retrieval</strong> of individual files by locating in which tar
+file a specific file is stored, as well as its location (offset) within the
+tar file.</p></li>
+<li><p><strong>File integrity</strong> is verified by computing checksums on-the-fly while <strong>extracting</strong>
+files.</p></li>
+</ul>
+<p>Source code is available on Github: <a class="reference external" href="https://github.com/E3SM-Project/zstash">https://github.com/E3SM-Project/zstash</a>.</p>
+<p>This documentation reflects the <code class="docutils literal notranslate"><span class="pre">v1.0.0</span></code> release of <code class="docutils literal notranslate"><span class="pre">zstash</span></code>.</p>
```

Comment:
Removed "`master` branch" and replaced with "v1.0.0 release".

Removed references to old builds since they link is outdated. Old builds are now stored in `_builds_old`, not `docs`.